modules/tiled-gallery/tiled-gallery.php

```diff
@@ -294,3 +294,6 @@ function setting_html() {
 }
 
 add_action( 'init', array( 'Jetpack_Tiled_Gallery', 'init' ) );
+add_action( 'init', function() {
```

Comment:
Before this could be merged, it would need to be converted to a function. Anonymous functions aren't allowed in PHP 5.2.